packages/common/utils/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.14.8",
+    "@toss/utility-types": "workspace:^1.0.5",
```

Comment:
This adds an additional dependency to our `utils` library (and even to users who do not use TypeScript), so I am a little bit concerned about if it is worth it. Why not just adding `arrayElements` to `utility-types`, rather than refactoring some of our code?

Reply:
Thank you for your kind words.
It could be a waste for people who don't use typescript!